src/guides/v2.4/graphql/interfaces/requisition-list-item-interface.md

```diff
@@ -0,0 +1,159 @@
+---
+group: graphql
+title: RequisitionListItemInterface attributes and implementations
```

Comment:
Does this need the `b2b_only: true` tag? The TOC entry has a B2B tag:https://github.com/magento/devdocs/pull/8428/files#diff-4d2753d92f3de59a53164caf094dc85f939ad66e8b834755573f26ea331d70bbR186

Reply:
Good catch. Thanks.